packages/design-tokens/less/typography.less

```diff
@@ -0,0 +1,43 @@
+/** THIS IS AN AUTOGENERATED FILE **/
+
+@kz-typography-heading0-fontFamily: "Greycliff CF";
```

Comment:
It still irks me that the case here is consistent

Reply:
We could change the script so that it compiles to kebab case for sass/less?